crates/locutus-core/src/ring.rs

```diff
@@ -405,6 +414,10 @@ impl Location {
             Location(1.0 - d)
         }
     }
+
+    pub fn as_f64(&self) -> f64 {
```

Comment:
Instead of operating directly on f64, I would suggest to impl  whatever arithmetic operations are needed for the types that need to compute something with a location. If we need we can make the internal f64 pub for now, since eventually the roter crate needs to be merged with this crate (actually we could implement the whole crate as a module of this crate if we want already just fine).

Reply:
Hmm, I think the issue here is that Distance was a typealias for Location - which doesn't seem right. I think I need to convert Distance to a struct and then add impls for the various required operations.

Reply:
@iduartgomez I made Distance its own struct rather than a Location - since distance and location are somewhat different concepts. Had to make some changes in locutus-core/src/node/test.rs and elsewhere - would you mind taking another look before I merge?